resolwe_bio/processes/variant_calling/ensembl_vep.py

```diff
@@ -0,0 +1,99 @@
+"""Run ensembl-vep."""
+
+from plumbum import TEE
+
+from resolwe.process import (
+    Cmd,
+    DataField,
+    FileField,
+    FileHtmlField,
+    Process,
+    SchedulingClass,
+)
+
+
+class EnsemblVep(Process):
+    """Run ensembl-vep."""
+
+    slug = "ensembl-vep"
+    name = "Ensembl Variant Effect Predictor"
+    category = "VEP"
+    process_type = "data:variants:vep"
+    version = "1.0.0"
+    scheduling_class = SchedulingClass.BATCH
+    entity = {"type": "sample"}
+    requirements = {
+        "expression-engine": "jinja",
+        "executor": {
+            "docker": {"image": "public.ecr.aws/genialis/resolwebio/dnaseq:6.2.0"}
+        },
+        "resources": {
+            "cores": 2,
+            "memory": 16384,
+        },
+    }
+    data_name = '{{vcf.file|default("?") }}'
+
+    class Input:
+        """Input fields for EnsemblVep."""
+
+        vcf = DataField(
+            "variants:vcf",
+            label="The main input, as produced in the GATK Genotype Refinement process",
+        )
+        cache = DataField("vep:cache", label="Cache directory for ensembl-vep")
+
+    class Output:
+        """Output fields for EnsemblVep."""
+
+        vcf = FileField(label="Refined multi-sample vcf")
+        tbi = FileField(label="Tabix index")
+        summary = FileHtmlField(label="Summary of the analysis")
+
+    def run(self, inputs, outputs):
+        """Run analysis."""
+
+        annotated_vcf = "snp_annotated.vcf"
+        annotated_vcf_gz = annotated_vcf + ".gz"
+        annotated_vcf_index = annotated_vcf_gz + ".tbi"
+        summary_html = annotated_vcf + "_summary.html"
+
+        if inputs.vcf.output.species != inputs.cache.output.species:
+            self.error(
+                "Species information for the input VCF file and cache don't match."
+            )
+        elif inputs.vcf.output.build != inputs.cache.output.build:
+            self.error(
+                "Build information for the input VCF file and cache don't match."
+            )
+
+        if inputs.cache.output.release != "104":
+            self.warning(
+                "The current version of ensembl-vep is 104. It is recommended that cache version is also 104"
```

Comment:
This will go out of sync as soon as we update the VEP tool version in Docker image. If the tool can return its version (e.g. --version, or -v, or ...), you should store this as a variable and use it instead of hardcoded value.